autoload.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace Onedrive;
```

Comment:
Let's please be consistent and use the same `namespace Krizalys\Onedrive;` everywhere.